packages/compiler-cli/src/ngtsc/typecheck/src/expression.ts

```diff
@@ -118,8 +120,13 @@ class AstTranslator implements AstVisitor {
     // Build up a chain of binary + operations to simulate the string concatenation of the
     // interpolation's expressions. The chain is started using an actual string literal to ensure
     // the type is inferred as 'string'.
+    // Each RHS expression needs to be wrapped in Parens so the trailing parse span info is not
+    // attributed to the whole binary expression. For example, a single interpolation without parens
+    // on the RHS would be `'"" + ((ctx).hello /*3,8*/) /*3,8*/`. The last 3,8 is ambiguous -- it
```

Comment:
```suggestion
    // on the RHS would be `"" + ((ctx).hello /*3,8*/) /*3,8*/`. The last 3,8 is ambiguous -- it
```


I can't think of a reason how this matters for diagnostics, as that is a bottom-up traversal and the statement itself should never produce diagnostics, as this is not assigned into anything? For the purpose of LS integration I can see how the top-down traversal it performs may need these parens, so in that context this change is still valid (although arguably not a fix, if my reasoning is correct here).